Pages/Sitemap.cshtml.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+using QCUtilities.Interfaces;
+using QCVault.Utilities.Entities;
+using QCVault.Utilities.Services;
+
+namespace QCVault.Pages
+{
+    public class SitemapModel : PageModel
+    {
+
+        private readonly ILogger<PageModel> logger;
+        private readonly IPostLoader postLoader;
+
+
+        public SitemapModel(ILogger<PageModel> logger, IPostLoader postLoader)
+        {
+            this.logger = logger;
+            this.postLoader = postLoader;
+        }
+
+        public IActionResult OnGet()
+        {
+            string baseUrl = "https://www.vault.themotte.org/";
```

Comment:
I'm not a big fan of this, just because of the whole url-hardcoding deal. I'm not sure there's a better solution right now, but at some point we probably need a config file or similar.